test/e2e/operator_test.go

```diff
@@ -1,4 +1,5 @@
-//+build e2e
+//go:build e2e
+// +build e2e
```

Comment:
yes these will be 1.17, i will remove this go build tag & verify
so as part of the build this `make verify` should fail ? am i right?

Reply:
Yes, it would fail if you used go1.17 on local and added this "new" tag because CI would try to build/test with go1.16 which would remove this tag and create `git diff`. `make verify` will verify for any unsolicited (not in PR) diffs.